wiki/Installation/pl.md

```diff
@@ -6,28 +6,28 @@ outdated: true
 
 *Zobacz także: [Instalacja/macOS](/wiki/Installation/macOS)*
 
-Ten artykuł przybliży ci proces instalacji osu! na urządzenia z systemem Windows. [Akademia osu!](/wiki/osu!academy) podjęła ten temat w swoim [poradniku wideo](https://www.youtube.com/watch?list=PLmWVQsxi34bMYwAawZtzuptfMmszUa_tl&v=cz522ZAs5aQ).
+Ten artykuł przybliży ci proces instalacji osu! na urządzenia z systemem Windows. [Akademia osu!](/wiki/osu!academy) podjęła ten temat w swoim [poradniku wideo](https://youtube.com/watch?v=0V5GwzmMhpU).
 
 ## Minimalne wymagania systemowe
 
-- .NET Framework 4+ (możesz znaleźć go [tutaj](https://www.microsoft.com/en-us/download/details.aspx?id=48130); instalator powininen automatycznie powiadomić cię o potrzebie zainstalowania tej platformy)
```

Comment:
what is this? why are you removing www from urls?

Reply:
because it's not needed in any way, so why should it be there?

Reply:
oh my god.

Reply:
in this case the canonical url does include www so you should keep it, you can tell bc without www redirects to www. sometimes the page includes a `<link rel="canonical" href="...">` to let you know the preferred url

but likewise you *shouldn't* include www if the service says it's not canonical